ui-smartlib-web/next.config.mjs

```diff
@@ -1,6 +1,8 @@
+import createNextIntlPlugin from 'next-intl/plugin';
+ 
+const withNextIntl = createNextIntlPlugin();
+ 
 /** @type {import('next').NextConfig} */
-const nextConfig = {
-    output: "standalone",
-};
```

Comment:
is there a reason why you removed this line ? "output: "standalone"

Reply:
nope. my bad, i might have removed it when adding NextIntl support. by mistake.